### PR TITLE
[Documentation] Update Skills section

### PR DESCRIPTION
**Pull request by Batta32**
Related to 3689

### Purpose
*What is the context of this pull request? Why is it being done?*
The [skills](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs/skills) section of the public documentation is outdated

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Migration from botbuilder-tools to botframework-cli commands
- Remove redundant information
- Fix broken/empty links
- Fix images
- Improve documentation

The updated documents are the followings:
- docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md
- docs/_docs/skills/handbook/best-practices.md
- docs/_docs/skills/handbook/botskills.md
- docs/_docs/skills/handbook/experimental-add-composer.md
- docs/_docs/skills/handbook/manifest.md
- docs/_docs/skills/samples/calendar.md
- docs/_docs/skills/tutorials/add-action-support/csharp/2-add-action.md
- docs/_docs/skills/tutorials/add-prebuilt-skill/1-intro.md
- docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md
- docs/_docs/skills/tutorials/create-skill/csharp/2-download-and-install.md
- docs/_docs/skills/tutorials/create-skill/csharp/3-create-your-skill.md
- docs/_docs/skills/tutorials/create-skill/csharp/4-provision-your-azure-resources.md
- docs/_docs/skills/tutorials/create-skill/typescript/2-download-and-install.md
- docs/_docs/skills/tutorials/create-skill/typescript/3-create-your-skill.md
- docs/_docs/skills/tutorials/create-skill/typescript/4-provision-your-azure-resources.md
- docs/_docs/skills/tutorials/customize-skill/csharp/2-edit-responses.md
- docs/_docs/skills/tutorials/customize-skill/typescript/2-edit-responses.md
- docs/_docs/skills/tutorials/customize-skill/typescript/2-edit-your-responses.md

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
It was rendered manually in order to check the result of the changes

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md [hunk 1]
```diff
@@ -13,96 +13,96 @@ toc: true
...
 
-## Updating an existing Skill to reflect changes to Actions or LUIS model
+> The _skillId_ parameter is case sensitive and can also be aquired using the `botskills list` command.
 
-To update a Skill to your assistant/Bot we provide a `botskills` command line tool to automate the process of adding the Skill to your dispatch model and creating authentication connections where needed.
+## Updating an existing Skill to reflect changes of the Skill Manifest
 
-Run the following command to update a Skill to your Virtual Assistant. This assumes you are running the CLI within the project directory and have created your Bot through the template and therefore have a `appsettings.json` file present in the working folder.
+Run the following command to update specific Skill to your Virtual Assistant. This assumes you are running the CLI within the Virtual Assistant's project directory and have created your bots through the template and therefore have a `appsettings.json` file present in the working folder.
```
Comment (FedericoBernal) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md:85]: shouldn't it be "Run the following command to update an specific Skill..."?

**Review by fran893 — APPROVED** [review]

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md (hunk 1 above)
Comment (Batta32) [reply docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md]: Done, thanks!

File: docs/_docs/skills/samples/calendar.md [hunk 2]
```diff
@@ -75,22 +75,23 @@ LUIS models for the Skill are provided in **.lu** file format as part of the Ski
...
+|FindCalendarWhere| Matches queries to get the location of a meeting|
+|FindCalendarWho| Matches queries to get the attendees of a meeting|
 |FindDuration| Matches queries to find out how long a meeting is|
 |FindMeetingRoom| Matches queries to find a meeting room|
-|GoBack| Matches queries to return to the previous step of a dialog|
-|NoLocation| Matches queries to not specify a location|
-|ReadAloud| Matches queries to read a calendar entry aloud |
+|RejectCalendar| Matches queries to reject an action of the calendar|
+|ShowNextCalendar| Matches queries to show the next events of the calendar|
+|ShowPreviousCalendar| Matches queries to show previous events of the calendar|
```
Comment (matiasroldan6) [inline docs/_docs/skills/samples/calendar.md:94]: these three lines look like quite a change. I'll assume this is ok

**Review by VictorGrycuk — APPROVED** [review]
Ship it

**Review by FedericoBernal — APPROVED** [review]
LGTM

**Review by matiasroldan6 — APPROVED** [review]

File: docs/_docs/skills/samples/calendar.md (hunk 2 above)
Comment (Batta32) [reply docs/_docs/skills/samples/calendar.md:94]: Yes, this is ok!

**Pull request merged**